test/run

```diff
@@ -11,7 +11,7 @@ if [ -z "${BASH_IT}" ]; then
 fi
 
 if [ -z "$1" ]; then
-  test_dirs=( ${test_directory}/{bash_it,completion,install,lib,plugins,themes} )
+  test_dirs=( "${test_directory}"/{bash_it,completion,install,lib,plugins,themes} )
```

Comment:
does this really belongs to this PR?

Reply:
I'm at a bit of a loss here, but I've removed it.